docs/source/performance/cache-backends.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: Configuring cache backends
+description: How to configure Apollo Server's cache
+---
+
+> ⚠️ New in Apollo Server 3.9: we **highly recommend** that all users pass `cache: "bounded"` or configure their cache in a manner which isn't unbounded (which is current default behavior) to protect your server from attacks which exhaust available memory causing a DOS. Continue reading for more information on configuring Apollo Server's cache.
```

Comment:
```suggestion
> ⚠️ **New in Apollo Server 3.9:** we _strongly recommend_ that all users pass `cache: "bounded"` or configure their cache in a manner that isn't unbounded (which is current default behavior). This protects your server from attacks that exhaust available memory, causing a DOS. Continue reading for more information on configuring Apollo Server's cache.
```

Reply:
Wasn't able to directly commit, but added via b30d11eac